common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -159,6 +159,7 @@ class @Problem
         @setupInputTypes()
         @bind()
         @queueing()
+        @renderProgressState()
```

Comment:
So I see that this is normally called in the other path via `forceUpdate`, and it makes sense we'd have to do this here. But I'm curious why the other lines in forceUpdate aren't necessary here as well:

```
    @el.data('progress_status', response.progress_status)
    @el.data('progress_detail', response.progress_detail)
    @el.trigger('progressChanged')
```